plugin/trino-kafka-event-listener/src/main/java/io/trino/plugin/eventlistener/kafka/RecordSerializer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.eventlistener.kafka;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.trino.plugin.eventlistener.kafka.model.QueryCompletedEventWrapper;
+import io.trino.plugin.eventlistener.kafka.model.QueryCreatedEventWrapper;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.kafka.common.serialization.Serializer;
+
+public class RecordSerializer
+        implements Serializer<Object>
+{
+    private final ObjectMapper mapper;
+
+    public RecordSerializer()
+    {
+        this.mapper = new ObjectMapper()
```

Comment:
We can also inject the ObjectMapper from the JsonCodec which we could use it here

Reply:
I removed this class `RecordSerializer` altogether. In the other class where ObjectMapper is used - `KafkaRecordBuilder` - I've changed to using `ObjectMapperProvider()` from airlift. (I wasn't able to pass in a property filter to `JsonCodec`)